MobileCenterDistribute/MobileCenterDistributeTests/MSDistributeTests.m

```diff
@@ -616,6 +616,8 @@ - (void)testWithoutNetwork {
   OCMStub([reachabilityMock reachabilityForInternetConnection]).andReturn(reachabilityMock);
   OCMStub([reachabilityMock currentReachabilityStatus]).andReturn(NotReachable);
   id distributeMock = OCMPartialMock(self.sut);
+  // We should not touch UI in a unit testing environment.
+  OCMStub([distributeMock requestUpdateToken:[OCMArg any]]).andDo(nil);
```

Comment:
Nope. If you need to mock to avoid touching UI, you should mock anything within the method.
Look at the below line. By adding this line, the test is meaningless.